scripts/ci/make/tasks

```diff
@@ -0,0 +1,155 @@
+        kind load docker-image "${engine_image}" --name e2e-testing
+    else
+        local engine_image="${DEV_IMAGE_REPO}:${COMMIT_SHA}"
+        print_colorized WARN "using remote dev image for testing -- ${engine_image}"
+    fi
+
+    local helm_deploy_name="e2e-testing-${RANDOM:0:5}"
+    helm repo add stable https://kubernetes-charts.storage.googleapis.com
+    helm install "${helm_deploy_name}" -f test/e2e/base-values.yaml --set anchoreGlobal.image="${engine_image}" stable/anchore-engine | head -n6
+
+    # Patch anchore-engine-api nodeport to the configured port thats mapped with kind to localhost:8228
+    kubectl patch service "${helm_deploy_name}-anchore-engine-api" --type=json --patch '[{"op": "replace", "path": "/spec/ports/0/nodePort", "value":30028}]'
+
+    install_testing_deps anchorecli
+    anchore-cli --u admin --p foobar system wait --timeout 300 --interval 10 --feedsready '' --servicesready 'catalog,apiext,policy_engine,simplequeue,analyzer'
+    anchore-cli --u admin --p foobar --url http://localhost:8228/v1 system status
+    "${PYTHON}" scripts/tests/aetest.py docker.io/alpine:latest
+    "${PYTHON}" scripts/tests/aefailtest.py docker.io/alpine:latest
+}
+
+push_rc_image() {
+    local dev_image="${DEV_IMAGE_REPO}:${COMMIT_SHA}"
+    local rc_image="${DEV_IMAGE_REPO}:${GIT_TAG}"
+
+    if [[ "${CI}" == true ]]; then
+        print_colorized WARN "pulling dev image for release candidate -- ${dev_image}"
+        docker pull "${dev_image}"
+
+        print_colorized WARN "tagging and pushing image -- ${rc_image}"
+        docker tag "${dev_image}" "${rc_image}"
+        docker push "${rc_image}"
+    else
+        print_colorized WARN "CI=true must be set to push image -- ${rc_image}"
+    fi
+}
+
+push_rebuild_image() {
+    local dev_image="${DEV_IMAGE_REPO}:${COMMIT_SHA}"
+    local rebuild_image="${PROD_IMAGE_REPO}:${GIT_TAG}"
+
+    if [[ "${CI}" == true ]]; then
+        print_colorized WARN "pulling dev image for release candidate -- ${dev_image}"
+        docker pull "${dev_image}"
+
+        print_colorized WARN "tagging and pushing image -- ${rebuild_image}"
+        sleep 3
+        docker tag "${dev_image}" "${rebuild_image}"
+        docker push "${rebuild_image}"
+    else
+        print_colorized WARN "CI=true must be set to push image -- ${rebuild_image}"
+    fi
+}
+
+push_release_image() {
+    local prod_image="${PROD_IMAGE_REPO}:${GIT_TAG}"
+    local rc_image="${DEV_IMAGE_REPO}:$(git describe --match "${GIT_TAG}-rc*" --tags --abbrev=0)"
+
+    if [[ "${CI}" == true ]]; then
+        print_colorized WARN "pulling RC image for release -- ${rc_image}"
+        docker pull "${rc_image}"
+
+        print_colorized WARN "tagging and pushing image -- ${prod_image}"
+        sleep 3
+        docker tag "${rc_image}" "${prod_image}"
+        docker push "${prod_image}"
+
+        if [[ "${GIT_BRANCH}" =~ "${LATEST_RELEASE_BRANCH}" ]]; then
```

Comment:
From Shellcheck (pycharm highlighted this for me): https://github.com/koalaman/shellcheck/wiki/SC2076